src/main/resources/assets/minelittlepony/lang/fr_fr.lang

```diff
@@ -10,10 +10,12 @@ minelp.options.hd=Activer MineLP serveur de skin
 minelp.options.sizes=Autoriser tous les diff�rentes tailles de poney
 minelp.options.ponyarmor=Utiliser armure compatible de MineLP
 minelp.options.snuzzles=Afficher museau sur les poneys
-minelp.options.showscale=Utiliser �chelle fid�le � MLP
```

Comment:
Minecraft doesn't render these characters, so we'd might as well switch to the non-tilda variants.

Reply:
Fix your encoding. Use UTF-8.

Reply:
@killjoy1221 I thought I was using uft8? The unchanged file still didn't render when reading the text in-game. I'll double-check.